### PR TITLE
All 1.6 changes

### PR DESCRIPTION
**Pull request by rspeer**
This update supersedes PR#46, and also includes a lot more changes.

The most notable change is that the `wordfreq_builder` subpackage has been taken out, and https://github.com/rspeer/exquisite-corpus is used instead. It's slightly more reproducible, and it increases the number of languages available and the number of sources available for most languages.

The big changes are listed in the CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -298,9 +312,9 @@ also try to deal gracefully when you query it with texts that actually break
 into multiple tokens:
 
     >>> zipf_frequency('New York', 'en')
-    5.07
+    5.31
```
Comment (alin-luminoso) [inline README.md]: Further on this topic -- I'm getting 5.35 here.  Did the data change?

File: wordfreq/transliterate.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# This table comes from https://github.com/opendatakosovo/cyrillic-transliteration/blob/master/cyrtranslit/mapping.py,
+# from the 'cyrtranslit' module, which is too flaky for us to just import.
```
Comment (alin-luminoso) [inline wordfreq/transliterate.py]: Given that this is for public release, I wonder if there's a less judgmental way to put this.

**Comment by alin-luminoso** [comment]
Re. removal of `wordfreq_builder`: should any of the material in `scripts/` have gone with that?

File: wordfreq/transliterate.py (hunk 2 above)
Comment (rspeer) [inline wordfreq/transliterate.py]: Sure, I'll get to the point: the cyrtranslit package is supposed to be compatible with Python 3, and its examples look like Python 3, but it uses import syntax that's Python 2 only. I've got a bug report open about that.

Using our own version of the table lets us handle mixed languages better, anyway.

File: README.md (hunk 1 above)
Comment (rspeer) [inline README.md]: I think I missed these examples the last time the data changed, yeah.

**Pull request merged**